easybuild/easyconfigs/c/custodian/custodian-1.1.0-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'Bundle'
+
+name = 'custodian'
+version = '1.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/custodian'
+description = """A simple JIT job management framework in Python."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('PyYAML', '3.12', versionsuffix),
+]
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+exts_list = [
+    ('custodian', version, {
+        'source_urls': ['https://pypi.python.org/packages/source/c/custodian/'],
+    }),
+    ('monty', '0.9.6', {
```

Comment:
`monty` a dependency for `custodian`, so it should be listed first (to avoid auto-installing it)